web/slashes/scrapers.py

```diff
@@ -0,0 +1,218 @@
+            return f"{formatting.bold(arg)}\n>{text_description}"
+        except Exception as e:
+            current_app.logger.exception(
+                f"Error while trying to find {self.word} {arg}: {e}"
+            )
+            return None
+
+
+class JobAdsCommandHandler(WebScrapingCommandHandler):
+    def __init__(self, *, relink_timeout):
+        super().__init__(relink_timeout=relink_timeout)
+
+        # @JoshK: Should these be arguments to the superclass constructor?
+        self.url = "https://htcondor.readthedocs.io/en/latest/classad-attributes/job-classad-attributes.html"
+        self.word = "job ad attribute"
+
+    def get_description(self, page_soup, arg):
+        try:
+            # This can't be the efficient way to do this.
+            spans = page_soup.select(
+                "dt > code.docutils.literal.notranslate > span.pre"
+            )
+
+            for span in spans:
+                if span.text.lower() == arg.lower():
+                    description = span.parent.parent.find_next("dd")
+                    for converter in [
+                        formatting.inplace_convert_em_to_underscores,
+                        formatting.inplace_convert_inline_code_to_backticks,
+                        formatting.inplace_convert_strong_to_stars,
+                        lambda soup: formatting.inplace_convert_internal_links_to_links(
+                            soup, os.path.dirname(self.url), "doc"
+                        ),
+                        formatting.inplace_convert_code_block_to_code_block,
+                    ]:
+                        converter(description)
+
+                    text_description = formatting.compress_whitespace(description.text)
+                    return f"{formatting.bold(span.text)}\n>{text_description}"
+            return None
+        except Exception as e:
+            current_app.logger.exception(
+                f"Error while trying to find {self.word} {arg}: {e}"
+            )
+            return None
+
+
+class SubmitsCommandHandler(WebScrapingCommandHandler):
+    def __init__(self, *, relink_timeout):
+        super().__init__(relink_timeout=relink_timeout)
+
+        # @JoshK: Should these be arguments to the superclass constructor?
+        self.url = (
+            "https://htcondor.readthedocs.io/en/latest/man-pages/condor_submit.html"
+        )
+        self.word = "submit file command"
+
+    def get_description(self, page_soup, arg):
+        try:
+            start = page_soup.find("div", id="submit-description-file-commands")
+
+            # No, we can't find_all_next("dt") and pass this regex to string,
+            # because some of our dt tags have children, and "text" isn't a
+            # keyword argument for some insane reason (it gets interpreted
+            # as an attribute search).
+            expr = re.compile(f"^{arg}( |$)", re.I)
+
+            # ... lambda?
```

Comment:
🤷‍♂️ I have no problem with named lambdas, but some people go crazy about it. Your preference.

Reply:
The question here wasn't if it should be a named lambda, but if it should be an inline one:

`dts = start.find_all_next(lambda tag: tag.name == "td" and expr.search(tag.text))`

(and why is GitHub making it hard to force that to be a single line)?

Reply:
Also no preference :)